addons/account/models/account_move.py

```diff
@@ -2918,6 +2918,40 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
+    def _post_check_invoice_non_negative(self):
+        """It checks that the total amount of a normal account cannot be less than zero.
+
+        In some accounting localizations this should be adjusted/disabled.
```

Comment:
> In some accounting localizations this should be disabled.

Apparently not since it is not used.

Reply:
For the Hungarian invoicing this should be disabled.